lib/templates.js

```diff
@@ -238,9 +238,48 @@ function _getPackageID(namespace, packageIds, resourceType) {
     return packageIds.find((id) => id.includes("-" + resourceType)) || packageIds.find((id) => id.includes(namespace));
 }
 
+const createIntegrationDependencyTemplate = (serviceName, serviceDefinition, appConfig, packageIds) => {
+
+    const ordExtensions = readORDExtensions(serviceDefinition);
+
+    const events = Object.keys(serviceDefinition.model.definitions)
+        .filter(event => {
+            const definition = serviceDefinition.model.definitions[event];
+            return definition["@topic"] && definition["@topic"].startsWith(serviceName) && !event.startsWith(serviceName);
+        })
+        .map(event => ({ eventType: serviceDefinition.model.definitions[event]["@topic"] }));
+
+    return {
```

Comment:
please follow the example of event and API templates to handle ordExtensions and don't explicitly access properties of ordExtensions and test that aspects can be overwritten as well (missing test)

Reply:
In the case of createIntegrationDependencyTemplate, in the ordExtensions, there is no attribute related to event, so how will we get the eventType for all the consumed external events?